mongomock/patch.py

```diff
@@ -0,0 +1,127 @@
+from .mongo_client import MongoClient
+from mongomock import InvalidURI
+import time
+
+try:
+    from unittest import mock
+    _IMPORT_MOCK_ERROR = None
+except ImportError:
+    try:
+        import mock
+        _IMPORT_MOCK_ERROR = None
+    except ImportError as error:
+        _IMPORT_MOCK_ERROR = error
+
+try:
+    import pymongo
+    _IMPORT_PYMONGO_ERROR = None
+except ImportError as error:
+    _IMPORT_PYMONGO_ERROR = error
+
+
+def patch(on_new='error', servers='localhost'):
```

Comment:
Maybe use `pymongo` as default `on_new`, so that the patch influences only the described servers